### PR TITLE
chore: bump core to v1.0.1

### PR DESCRIPTION
**Pull request by liamsi**

### CODE REVIEW

**Comment by adlerjohn** [comment]
Converting to draft until @evan-forbes fixes the flaky test

**Comment by liamsi** [comment]
https://github.com/celestiaorg/celestia-core/issues/718

**Comment by evan-forbes** [comment]
sorry, didn't mean to push directly to this branch, thought I was on a different branch. It should work tho. If we want to take a different approach, then I can fix the commit :slightly_smiling_face:

File: x/auth/client/testutil/suite.go [hunk 1]
```diff
@@ -461,6 +462,8 @@ func (s *IntegrationTestSuite) TestCLIQueryTxsCmdByEvents() {
 				fmt.Sprintf("--%s=json", tmcli.OutputFlag),
 			},
 			false,
+			"transaction spanned more than two shares, this is not yet supported",
+			// TODO: change this to not expect an error when functionality is added to celestia-core
```
Comment (liamsi) [inline x/auth/client/testutil/suite.go:466]: Should we track this in an issue instead?

Comment (liamsi) [reply x/auth/client/testutil/suite.go:466]: Or, actually this is sufficient I think: https://github.com/celestiaorg/celestia-core/issues/718#issuecomment-1098158995

**Pull request merged**